spec/requests/api/v2/interrupt_page_acknowlegements_spec.rb

```diff
@@ -0,0 +1,96 @@
+
+require "rails_helper"
+
+describe "Interrupt page acknowledgements API v2", type: :request do
+  let(:user)         { create(:user) }
+  let(:cycle)        { find_or_create :recruitment_cycle }
+  let(:payload)      { { email: user.email } }
+  let(:credentials) { encode_to_credentials(payload) }
+
+  describe "index" do
+    before do
+      create(:interrupt_page_acknowledgement, user: user, recruitment_cycle: cycle, page: :rollover)
```

Comment:
```suggestion
      create(:interrupt_page_acknowledgement, user: user, recruitment_cycle: cycle)
```

Reply:
Id rather keep this here to make the test more obvious than relying on the default factory behaviour